api/src/routes/events.ts

```diff
@@ -70,8 +79,7 @@ eventRouter.put("/", [auth], async (req: Request, res: Response) => {
   }
 
   const eventId = req.body.id;
-  const id = (req as AuthRequest).user.id;
```

Comment:
is line 73 supposed to be removed?

Reply:
Not sure, but I dont think so? @zusoomro

Reply:
other than this, I think it looks great! Also, it looks like the id variable from line 73 is never used so it's probably unnecessary

Reply:
Yeah it was just something random that was throwing an error in my ide so i removed it whooops